pkg/search/result.go

```diff
@@ -95,17 +95,16 @@ type Repository struct {
 	PushedAt        time.Time `json:"pushed_at"`
 	Size            int       `json:"size"`
 	StargazersCount int       `json:"stargazers_count"`
-	URL             string    `json:"url"`
+	URL             string    `json:"html_url"`
 	UpdatedAt       time.Time `json:"updated_at"`
 	Visibility      string    `json:"visibility"`
 	WatchersCount   int       `json:"watchers_count"`
 }
 
 type License struct {
-	HTMLURL string `json:"html_url"`
-	Key     string `json:"key"`
-	Name    string `json:"name"`
-	URL     string `json:"url"`
+	Key  string `json:"key"`
+	Name string `json:"name"`
+	URL  string `json:"html_url"`
```

Comment:
It doesn't seem like the license output from Search API includes `html_url` in its JSON; just API `url`.